apps/provider-console/src/components/become-provider/ServerAccess.tsx

```diff
@@ -0,0 +1,55 @@
+"use client";
+import React, { useCallback, useState } from "react";
+import { Button, Input, Separator } from "@akashnetwork/ui/components";
+
+import { ResetProviderForm } from "./ResetProviderProcess";
+import { ServerForm } from "./ServerForm";
+
+interface ServerAccessProps {
+  stepChange: () => void;
```

Comment:
in this case we should care about interface and client code. E.g.

```tsx
<ServerAccess stepChange={doSmth} />
```

this code is misleading as it reads like a component needs a function to change a step. This might be the case but then it kinda verbally breaks the separation of concerns. Instead it should be an event like callback.

```tsx
<ServerAccess onSmthHappend={changeStep} />
```
```tsx
<ServerAccess onSubmit={changeStep} />
```

Reply:
changed to onComplete on all of the actions.